meson.build

```diff
@@ -0,0 +1,23 @@
+project('PulseEffects',
+  version: '2.3.3'    
+)
+
+py3_mod = import('python3')
+i18n_mod = import('i18n')
+
+message('Checking dependencies')
```

Comment:
I think these messages are pretty silly, its not like any of these actions can fail.

Reply:
Well it is my first time using Meson. I did not fully understand what the messages was for. Part of me had them there to act as comments.